witchcraft-logging-idea/src/test/java/com/palantir/witchcraft/logging/idea/WitchcraftLogFormatFilterTest.java

```diff
@@ -38,6 +38,10 @@
             + "\"time\":\"2019-05-09T15:32:37.692Z\",\"origin\":\"ROOT\","
             + "\"thread\":\"main\",\"message\":\"test good {}\","
             + "\"params\":{\"good\":\":-)\"},\"unsafeParams\":{},\"tags\":{}}";
+    private static final String SERVICE_JSON_WITH_UNKNOWN_FIELD = "{\"type\":\"service.1\",\"level\":\"ERROR\","
+            + "\"time\":\"2019-05-09T15:32:37.692Z\",\"origin\":\"ROOT\","
+            + "\"thread\":\"main\",\"message\":\"test good {}\","
+            + "\"params\":{\"good\":\":-)\"},\"unsafeParams\":{},\"unknownField\":\"value\"}";
```

Comment:
duplicate test data -- eventually I'll probably split out the parsing/formatting code from the idea plugin module.